[2025-01-01 22:18:16,424: INFO: main: Welcome to our custom log]
[2025-01-02 00:14:57,192: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:17:10,860: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:17:10,901: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:17:10,914: INFO: common: created directory at: artifacts]
[2025-01-02 00:19:36,410: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:19:36,428: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:19:36,438: INFO: common: created directory at: artifacts]
[2025-01-02 00:20:14,035: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:20:14,186: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:20:14,198: INFO: common: created directory at: artifacts]
[2025-01-02 00:30:16,777: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:30:16,798: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:30:16,809: INFO: common: created directory at: artifacts]
[2025-01-02 00:30:16,816: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:30:16,820: INFO: 3452066675: fDownloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing inot file artifacts/data_ingestion/data.zip]
[2025-01-02 00:30:25,104: INFO: 3452066675: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:33:27,634: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:33:27,657: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:33:27,663: INFO: common: created directory at: artifacts]
[2025-01-02 00:33:27,671: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:33:27,676: INFO: 4186505339: fDownloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing inot file artifacts/data_ingestion/data.zip]
[2025-01-02 00:33:35,915: INFO: 4186505339: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:41:23,797: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:41:23,806: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:41:23,809: INFO: common: created directory at: artifacts]
[2025-01-02 00:41:23,813: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:41:23,816: INFO: 3768243799: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:41:30,680: INFO: 3768243799: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:43:46,898: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:43:46,908: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:43:46,913: INFO: common: created directory at: artifacts]
[2025-01-02 00:43:46,916: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:43:46,923: INFO: 3768243799: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:43:53,268: INFO: 3768243799: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:46:26,265: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:46:26,274: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:46:26,278: INFO: common: created directory at: artifacts]
[2025-01-02 00:46:26,288: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:46:26,296: INFO: 3768243799: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:46:32,223: INFO: 3768243799: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:49:16,709: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:49:16,721: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:49:16,736: INFO: common: created directory at: artifacts]
[2025-01-02 00:49:16,743: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:49:16,816: INFO: 1535160724: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:49:24,614: INFO: 1535160724: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:53:07,716: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:53:07,726: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:53:07,732: INFO: common: created directory at: artifacts]
[2025-01-02 00:53:07,738: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:53:07,745: INFO: 3628025189: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:53:14,371: INFO: 3628025189: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:55:43,468: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-02 00:55:43,501: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-02 00:55:43,517: INFO: common: created directory at: artifacts]
[2025-01-02 00:55:43,529: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-02 00:55:43,535: INFO: 3628025189: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 00:55:50,986: INFO: 3628025189: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-02 20:53:26,171: INFO: main: Welcome to our custom log]
[2025-01-03 01:28:53,309: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-03 01:28:53,327: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-03 01:28:53,329: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-03 01:28:53,331: INFO: common: created directory at: artifacts]
[2025-01-03 01:28:53,332: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-03 01:28:53,333: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-03 01:29:01,048: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-03 01:29:02,993: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-03 01:29:38,198: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-03 01:29:38,203: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-03 01:29:38,205: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-03 01:29:38,206: INFO: common: created directory at: artifacts]
[2025-01-03 01:29:38,206: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-03 01:29:38,206: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-03 01:29:44,691: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-03 01:29:46,064: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-03 01:44:40,732: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-03 01:44:40,744: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-03 01:44:40,754: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-03 01:44:40,762: INFO: common: created directory at: artifacts]
[2025-01-03 01:44:40,763: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-03 01:44:40,768: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-03 01:44:47,227: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-03 01:44:48,393: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-04 20:56:00,870: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-04 20:56:00,875: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 20:56:00,876: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 20:56:00,876: INFO: common: created directory at: artifacts]
[2025-01-04 20:56:00,877: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-04 20:56:00,877: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-04 20:56:09,804: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-04 20:56:10,887: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-04 22:01:40,618: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:01:40,865: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:01:40,956: INFO: common: created directory at: artifacts]
[2025-01-04 22:04:32,967: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:04:32,993: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:04:33,002: INFO: common: created directory at: artifacts]
[2025-01-04 22:05:13,938: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:05:13,992: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:05:14,013: INFO: common: created directory at: artifacts]
[2025-01-04 22:06:37,346: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:06:37,376: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:06:37,489: INFO: common: created directory at: artifacts]
[2025-01-04 22:06:37,498: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:08:34,369: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:08:34,393: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:08:34,400: INFO: common: created directory at: artifacts]
[2025-01-04 22:08:34,416: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:12:37,302: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:12:37,343: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:12:37,371: INFO: common: created directory at: artifacts]
[2025-01-04 22:12:37,374: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:13:32,327: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:13:32,406: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:13:32,421: INFO: common: created directory at: artifacts]
[2025-01-04 22:13:32,429: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:16:29,747: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:16:29,856: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:16:29,872: INFO: common: created directory at: artifacts]
[2025-01-04 22:16:29,881: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:20:13,822: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:20:13,886: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:20:13,904: INFO: common: created directory at: artifacts]
[2025-01-04 22:20:13,914: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:23:07,469: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:23:07,488: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:23:07,501: INFO: common: created directory at: artifacts]
[2025-01-04 22:23:07,506: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:26:06,289: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:26:06,339: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:26:06,350: INFO: common: created directory at: artifacts]
[2025-01-04 22:26:06,363: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:26:20,636: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 22:29:20,972: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:29:21,078: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:29:21,095: INFO: common: created directory at: artifacts]
[2025-01-04 22:29:21,115: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:29:26,140: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 22:31:35,667: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:31:35,762: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:31:35,766: INFO: common: created directory at: artifacts]
[2025-01-04 22:31:35,769: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:31:37,416: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 22:37:09,481: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:37:09,533: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:37:09,547: INFO: common: created directory at: artifacts]
[2025-01-04 22:37:09,574: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:37:13,051: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 22:38:31,273: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:38:31,288: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:38:31,299: INFO: common: created directory at: artifacts]
[2025-01-04 22:38:31,312: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:38:34,096: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 22:39:37,593: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:39:37,713: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:39:37,730: INFO: common: created directory at: artifacts]
[2025-01-04 22:39:37,737: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 22:39:39,109: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 22:44:13,237: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-04 22:44:13,244: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 22:44:13,249: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 22:44:13,249: INFO: common: created directory at: artifacts]
[2025-01-04 22:44:13,250: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-04 22:44:13,250: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-04 22:44:23,877: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-04 22:44:25,596: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-04 23:13:29,801: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-04 23:13:29,805: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 23:13:29,816: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 23:13:29,820: INFO: common: created directory at: artifacts]
[2025-01-04 23:13:29,820: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-04 23:13:29,821: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-04 23:13:35,633: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-04 23:13:37,608: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-04 23:13:37,609: INFO: main: *******************]
[2025-01-04 23:13:37,609: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-04 23:13:37,621: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-04 23:13:37,636: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-04 23:13:37,637: INFO: common: created directory at: artifacts]
[2025-01-04 23:13:37,637: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-04 23:13:39,696: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-04 23:13:41,112: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-05 12:34:12,838: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-05 12:34:12,845: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 12:34:12,855: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 12:34:12,859: INFO: common: created directory at: artifacts]
[2025-01-05 12:34:12,860: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-05 12:34:12,860: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 12:34:20,754: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 12:34:22,536: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-05 12:34:22,536: INFO: main: *******************]
[2025-01-05 12:34:22,536: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-05 12:34:22,542: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 12:34:22,549: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 12:34:22,550: INFO: common: created directory at: artifacts]
[2025-01-05 12:34:22,552: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-05 12:34:24,047: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-05 12:34:25,108: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-05 14:04:02,527: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:04:02,674: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:04:02,961: INFO: common: created directory at: artifacts]
[2025-01-05 14:05:53,708: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:05:53,826: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:05:53,861: INFO: common: created directory at: artifacts]
[2025-01-05 14:05:53,916: INFO: common: created directory at: artifacts/training]
[2025-01-05 14:07:04,145: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:07:04,174: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:07:04,315: INFO: common: created directory at: artifacts]
[2025-01-05 14:07:04,332: INFO: common: created directory at: artifacts/training]
[2025-01-05 14:07:55,871: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:07:55,896: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:07:55,909: INFO: common: created directory at: artifacts]
[2025-01-05 14:07:55,914: INFO: common: created directory at: artifacts/training]
[2025-01-05 14:12:09,291: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:12:09,571: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:12:09,628: INFO: common: created directory at: artifacts]
[2025-01-05 14:12:09,650: INFO: common: created directory at: artifacts/training]
[2025-01-05 14:15:40,191: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:15:40,209: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:15:40,215: INFO: common: created directory at: artifacts]
[2025-01-05 14:15:40,221: INFO: common: created directory at: artifacts/training]
[2025-01-05 14:51:07,662: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-05 14:51:07,678: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:51:07,686: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:51:07,691: INFO: common: created directory at: artifacts]
[2025-01-05 14:51:07,691: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-05 14:51:07,692: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 14:51:16,249: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 14:51:17,799: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-05 14:51:17,800: INFO: main: *******************]
[2025-01-05 14:51:17,800: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-05 14:51:17,815: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:51:17,818: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:51:17,819: INFO: common: created directory at: artifacts]
[2025-01-05 14:51:17,820: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-05 14:51:20,244: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-05 14:51:22,158: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-05 14:51:22,159: INFO: main: *******************]
[2025-01-05 14:51:22,160: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-05 14:51:22,167: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:51:22,170: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:51:22,171: INFO: common: created directory at: artifacts]
[2025-01-05 14:51:22,171: ERROR: main: 'ConfiguartionManager' object has no attribute 'get_training_config_config']
Traceback (most recent call last):
  File "main.py", line 39, in <module>
    model_trainer.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_03_model_training.py", line 15, in main
    training_config = config.get_training_config_config()
AttributeError: 'ConfiguartionManager' object has no attribute 'get_training_config_config'
[2025-01-05 14:53:23,315: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-05 14:53:23,320: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:53:23,329: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:53:23,334: INFO: common: created directory at: artifacts]
[2025-01-05 14:53:23,334: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-05 14:53:23,335: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 14:53:30,985: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 14:53:34,647: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-05 14:53:34,648: INFO: main: *******************]
[2025-01-05 14:53:34,648: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-05 14:53:34,661: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:53:34,670: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:53:34,673: INFO: common: created directory at: artifacts]
[2025-01-05 14:53:34,677: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-05 14:53:37,160: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-05 14:53:38,167: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-05 14:53:38,167: INFO: main: *******************]
[2025-01-05 14:53:38,168: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-05 14:53:38,175: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 14:53:38,178: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 14:53:38,179: INFO: common: created directory at: artifacts]
[2025-01-05 14:53:38,180: INFO: common: created directory at: artifacts/training]
[2025-01-05 14:57:25,495: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-05 15:30:23,526: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-05 15:30:23,534: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 15:30:23,545: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 15:30:23,548: INFO: common: created directory at: artifacts]
[2025-01-05 15:30:23,548: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-05 15:30:23,549: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 15:30:29,713: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-05 15:30:30,964: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-05 15:30:30,965: INFO: main: *******************]
[2025-01-05 15:30:30,966: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-05 15:30:30,970: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 15:30:30,973: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 15:30:30,974: INFO: common: created directory at: artifacts]
[2025-01-05 15:30:30,974: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-05 15:30:32,425: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-05 15:30:33,423: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-05 15:30:33,423: INFO: main: *******************]
[2025-01-05 15:30:33,424: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-05 15:30:33,433: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-05 15:30:33,438: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-05 15:30:33,439: INFO: common: created directory at: artifacts]
[2025-01-05 15:30:33,440: INFO: common: created directory at: artifacts/training]
[2025-01-06 00:38:08,523: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 00:38:08,733: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 00:38:08,784: INFO: common: created directory at: artifacts]
[2025-01-06 00:42:19,392: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 00:42:19,563: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 00:42:19,572: INFO: common: created directory at: artifacts]
[2025-01-06 00:44:47,943: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 00:44:48,019: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 00:44:48,031: INFO: common: created directory at: artifacts]
[2025-01-06 00:47:55,107: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 00:47:55,157: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 00:47:55,229: INFO: common: created directory at: artifacts]
[2025-01-06 00:49:54,961: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 00:49:54,987: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 00:49:54,994: INFO: common: created directory at: artifacts]
[2025-01-06 01:22:41,184: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 01:22:41,217: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 01:22:41,246: INFO: common: created directory at: artifacts]
[2025-01-06 01:27:15,049: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 01:27:15,079: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 01:27:21,743: INFO: common: created directory at: artifacts]
[2025-01-06 01:32:36,784: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 01:32:36,870: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 01:32:36,879: INFO: common: created directory at: artifacts]
[2025-01-06 01:48:59,441: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 01:48:59,515: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 01:48:59,563: INFO: common: created directory at: artifacts]
[2025-01-06 01:53:59,190: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 01:53:59,229: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 01:53:59,245: INFO: common: created directory at: artifacts]
[2025-01-06 01:55:23,864: INFO: common: json file saved at scores.json]
[2025-01-06 01:58:25,117: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 01:58:25,146: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 01:58:25,165: INFO: common: created directory at: artifacts]
[2025-01-06 02:01:03,313: INFO: common: json file saved at scores.json]
[2025-01-06 02:13:04,238: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 02:13:04,286: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 02:13:04,325: INFO: common: created directory at: artifacts]
[2025-01-06 02:14:41,447: INFO: common: json file saved at scores.json]
[2025-01-06 02:35:04,967: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 02:35:05,050: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 02:35:05,057: INFO: common: created directory at: artifacts]
[2025-01-06 02:37:07,462: INFO: common: json file saved at scores.json]
[2025-01-06 02:46:44,790: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 02:46:44,969: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 02:46:44,978: INFO: common: created directory at: artifacts]
[2025-01-06 02:48:24,132: INFO: common: json file saved at scores.json]
[2025-01-06 02:48:38,752: INFO: _client: HTTP Request: POST https://dagshub.com/login/oauth/middleman "HTTP/1.1 200 OK"]
[2025-01-06 02:48:39,731: INFO: _client: HTTP Request: POST https://dagshub.com/login/oauth/access_token "HTTP/1.1 200 OK"]
[2025-01-06 02:48:40,598: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-06 02:48:40,656: INFO: helpers: Accessing as Calivala-86]
[2025-01-06 02:48:41,035: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-06 02:48:41,448: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-06 02:48:41,459: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-06 02:48:41,569: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-06 02:48:48,002: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 02:48:52,086: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpwl5gvfgt/model/data/model/assets]
[2025-01-06 03:04:00,588: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 03:04:00,714: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 03:04:00,729: INFO: common: created directory at: artifacts]
[2025-01-06 03:06:21,308: INFO: common: json file saved at scores.json]
[2025-01-06 03:06:25,745: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 03:06:28,577: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmp7m__16aa/model/data/model/assets]
[2025-01-06 03:14:53,600: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 03:14:53,621: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 03:14:53,638: INFO: common: created directory at: artifacts]
[2025-01-06 03:16:16,634: INFO: common: json file saved at scores.json]
[2025-01-06 03:16:22,053: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 03:16:24,149: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpava_jfk3/model/data/model/assets]
[2025-01-06 20:43:29,523: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 20:43:29,636: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 20:43:29,649: INFO: common: created directory at: artifacts]
[2025-01-06 20:45:07,061: INFO: common: json file saved at scores.json]
[2025-01-06 20:45:13,198: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 20:45:15,321: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpv5mt9_fk/model/data/model/assets]
[2025-01-06 20:46:56,721: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 20:46:56,737: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 20:46:56,743: INFO: common: created directory at: artifacts]
[2025-01-06 20:48:33,096: INFO: common: json file saved at scores.json]
[2025-01-06 20:48:39,273: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 20:48:41,300: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpp_hd5exk/model/data/model/assets]
[2025-01-06 21:52:41,995: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-06 21:52:42,002: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 21:52:42,014: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 21:52:42,020: INFO: common: created directory at: artifacts]
[2025-01-06 21:52:42,021: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-06 21:52:42,021: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 21:52:49,213: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 21:52:50,367: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-06 21:52:50,368: INFO: main: *******************]
[2025-01-06 21:52:50,368: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-06 21:52:50,382: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 21:52:50,386: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 21:52:50,387: INFO: common: created directory at: artifacts]
[2025-01-06 21:52:50,387: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-06 21:52:51,955: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-06 21:52:53,549: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-06 21:52:53,550: INFO: main: *******************]
[2025-01-06 21:52:53,550: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-06 21:52:53,555: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 21:52:53,558: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 21:52:53,559: INFO: common: created directory at: artifacts]
[2025-01-06 21:52:53,560: INFO: common: created directory at: artifacts/training]
[2025-01-06 21:56:23,905: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-06 21:56:23,920: INFO: main: *******************]
[2025-01-06 21:56:23,920: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-06 21:56:23,936: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 21:56:23,947: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 21:56:23,954: INFO: common: created directory at: artifacts]
[2025-01-06 21:57:26,950: INFO: common: json file saved at scores.json]
[2025-01-06 21:57:26,955: INFO: common: json file saved at scores.json]
[2025-01-06 21:57:31,049: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 21:57:32,364: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpye6e5rim/model/data/model/assets]
[2025-01-06 21:57:59,862: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "main.py", line 53, in <module>
    model_evaluation.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 19, in main
    evaluation.log_into_mlflow()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/components/model_evaluation_mlflow.py", line 69, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tensorflow/__init__.py", line 222, in log_model
    return Model.log(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/models/model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/_tracking_service/client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/store/artifact/local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2025-01-06 22:06:28,997: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-06 22:06:29,002: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:06:29,011: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:06:29,017: INFO: common: created directory at: artifacts]
[2025-01-06 22:06:29,017: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-06 22:06:29,017: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:06:36,472: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:06:37,443: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-06 22:06:37,443: INFO: main: *******************]
[2025-01-06 22:06:37,443: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-06 22:06:37,448: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:06:37,451: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:06:37,452: INFO: common: created directory at: artifacts]
[2025-01-06 22:06:37,452: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-06 22:06:38,896: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-06 22:06:40,295: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-06 22:06:40,296: INFO: main: *******************]
[2025-01-06 22:06:40,296: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-06 22:06:40,305: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:06:40,311: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:06:40,312: INFO: common: created directory at: artifacts]
[2025-01-06 22:06:40,313: INFO: common: created directory at: artifacts/training]
[2025-01-06 22:10:04,703: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-06 22:10:04,709: INFO: main: *******************]
[2025-01-06 22:10:04,709: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-06 22:10:04,723: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:10:04,733: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:10:04,740: INFO: common: created directory at: artifacts]
[2025-01-06 22:10:55,363: INFO: common: json file saved at scores.json]
[2025-01-06 22:10:55,367: INFO: common: json file saved at scores.json]
[2025-01-06 22:10:59,571: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 22:11:00,888: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpai1fi3bw/model/data/model/assets]
[2025-01-06 22:11:29,235: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "main.py", line 53, in <module>
    model_evaluation.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 19, in main
    evaluation.log_into_mlflow()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/components/model_evaluation_mlflow.py", line 69, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tensorflow/__init__.py", line 222, in log_model
    return Model.log(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/models/model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/_tracking_service/client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/store/artifact/local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2025-01-06 22:23:14,954: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-06 22:23:14,960: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:23:14,974: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:23:14,980: INFO: common: created directory at: artifacts]
[2025-01-06 22:23:14,981: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-06 22:23:14,981: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:23:22,161: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:23:23,824: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-06 22:23:23,824: INFO: main: *******************]
[2025-01-06 22:23:23,824: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-06 22:23:23,832: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:23:23,836: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:23:23,836: INFO: common: created directory at: artifacts]
[2025-01-06 22:23:23,836: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-06 22:23:25,519: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-06 22:23:26,553: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-06 22:23:26,553: INFO: main: *******************]
[2025-01-06 22:23:26,553: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-06 22:23:26,560: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:23:26,563: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:23:26,564: INFO: common: created directory at: artifacts]
[2025-01-06 22:23:26,564: INFO: common: created directory at: artifacts/training]
[2025-01-06 22:26:31,512: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-06 22:26:31,522: INFO: main: *******************]
[2025-01-06 22:26:31,523: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-06 22:26:31,536: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:26:31,546: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:26:31,554: INFO: common: created directory at: artifacts]
[2025-01-06 22:27:23,152: INFO: common: json file saved at scores.json]
[2025-01-06 22:27:23,155: INFO: common: json file saved at scores.json]
[2025-01-06 22:27:27,414: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 22:27:28,620: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmp1kuiu_3w/model/data/model/assets]
[2025-01-06 22:27:56,837: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "main.py", line 53, in <module>
    model_evaluation.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 19, in main
    evaluation.log_into_mlflow()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/components/model_evaluation_mlflow.py", line 69, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tensorflow/__init__.py", line 222, in log_model
    return Model.log(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/models/model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/_tracking_service/client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/store/artifact/local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2025-01-06 22:41:46,287: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-06 22:41:46,293: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:41:46,304: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:41:46,308: INFO: common: created directory at: artifacts]
[2025-01-06 22:41:46,308: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-06 22:41:46,309: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:41:52,504: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:41:53,727: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-06 22:41:53,727: INFO: main: *******************]
[2025-01-06 22:41:53,727: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-06 22:41:53,731: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:41:53,734: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:41:53,734: INFO: common: created directory at: artifacts]
[2025-01-06 22:41:53,735: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-06 22:41:55,005: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-06 22:41:56,098: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-06 22:41:56,098: INFO: main: *******************]
[2025-01-06 22:41:56,099: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-06 22:41:56,105: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:41:56,109: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:41:56,109: INFO: common: created directory at: artifacts]
[2025-01-06 22:41:56,110: INFO: common: created directory at: artifacts/training]
[2025-01-06 22:46:01,359: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-06 22:46:01,414: INFO: main: *******************]
[2025-01-06 22:46:01,414: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-06 22:46:01,476: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:46:01,545: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:46:01,573: INFO: common: created directory at: artifacts]
[2025-01-06 22:49:57,538: INFO: common: json file saved at scores.json]
[2025-01-06 22:49:57,540: INFO: common: json file saved at scores.json]
[2025-01-06 22:50:10,317: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 22:50:14,239: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmp46sw2veb/model/data/model/assets]
[2025-01-06 22:50:50,619: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "main.py", line 53, in <module>
    model_evalution.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 19, in main
    evaluation.log_into_mlflow()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/components/model_evaluation_mlflow.py", line 69, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tensorflow/__init__.py", line 222, in log_model
    return Model.log(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/models/model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/_tracking_service/client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/store/artifact/local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2025-01-06 22:51:54,932: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-06 22:51:54,938: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:51:54,948: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:51:54,951: INFO: common: created directory at: artifacts]
[2025-01-06 22:51:54,952: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-06 22:51:54,952: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:52:01,867: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-06 22:52:03,723: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-06 22:52:03,723: INFO: main: *******************]
[2025-01-06 22:52:03,723: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-06 22:52:03,739: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:52:03,748: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:52:03,748: INFO: common: created directory at: artifacts]
[2025-01-06 22:52:03,749: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-06 22:52:05,408: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-06 22:52:09,719: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-06 22:52:09,819: INFO: main: *******************]
[2025-01-06 22:52:09,843: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-06 22:52:10,102: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:52:10,109: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:52:10,112: INFO: common: created directory at: artifacts]
[2025-01-06 22:52:10,113: INFO: common: created directory at: artifacts/training]
[2025-01-06 22:57:43,731: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-06 22:57:43,738: INFO: main: *******************]
[2025-01-06 22:57:43,739: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-06 22:57:43,763: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-06 22:57:43,837: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-06 22:57:43,841: INFO: common: created directory at: artifacts]
[2025-01-06 22:59:06,096: INFO: common: json file saved at scores.json]
[2025-01-06 22:59:06,102: INFO: common: json file saved at scores.json]
[2025-01-06 22:59:10,785: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-06 22:59:12,446: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmptshmlzbh/model/data/model/assets]
[2025-01-06 22:59:38,882: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "main.py", line 53, in <module>
    model_evalution.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 19, in main
    evaluation.log_into_mlflow()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/components/model_evaluation_mlflow.py", line 69, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tensorflow/__init__.py", line 222, in log_model
    return Model.log(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/models/model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/tracking/_tracking_service/client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/mlflow/store/artifact/local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/setuptools/_distutils/file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2025-01-09 23:12:16,698: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-09 23:12:16,716: INFO: helpers: Accessing as Calivala-86]
[2025-01-09 23:12:17,686: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-09 23:12:17,960: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-09 23:12:17,967: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-09 23:12:17,971: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-09 23:12:29,219: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-09 23:12:29,229: INFO: helpers: Accessing as Calivala-86]
[2025-01-09 23:12:29,509: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-09 23:12:29,973: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-09 23:12:29,977: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-09 23:12:29,978: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-09 23:12:29,979: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-09 23:12:29,983: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-09 23:12:29,990: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-09 23:12:29,993: INFO: common: created directory at: artifacts]
[2025-01-09 23:12:29,993: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-09 23:12:29,994: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-09 23:12:37,575: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-09 23:12:39,084: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-09 23:12:39,087: INFO: main: *******************]
[2025-01-09 23:12:39,087: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-09 23:12:39,093: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-09 23:12:39,096: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-09 23:12:39,097: INFO: common: created directory at: artifacts]
[2025-01-09 23:12:39,097: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-09 23:12:40,902: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-09 23:12:42,058: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-09 23:12:42,058: INFO: main: *******************]
[2025-01-09 23:12:42,059: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-09 23:12:42,064: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-09 23:12:42,069: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-09 23:12:42,070: INFO: common: created directory at: artifacts]
[2025-01-09 23:12:42,070: INFO: common: created directory at: artifacts/training]
[2025-01-09 23:16:11,804: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-09 23:16:11,812: INFO: main: *******************]
[2025-01-09 23:16:11,812: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-09 23:16:11,833: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-09 23:16:11,846: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-09 23:16:11,856: INFO: common: created directory at: artifacts]
[2025-01-09 23:17:05,029: INFO: common: json file saved at scores.json]
[2025-01-09 23:17:05,031: INFO: common: json file saved at scores.json]
[2025-01-09 23:17:11,277: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-09 23:17:12,582: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpygk0nl4a/model/data/model/assets]
[2025-01-09 23:17:59,573: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-01-09 23:59:49,415: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-09 23:59:49,538: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-09 23:59:49,595: INFO: common: created directory at: artifacts]
[2025-01-10 00:03:04,479: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 00:03:04,513: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 00:03:04,540: INFO: common: created directory at: artifacts]
[2025-01-10 00:04:18,116: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 00:04:18,238: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 00:04:18,252: INFO: common: created directory at: artifacts]
[2025-01-10 00:07:11,807: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 00:07:11,825: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 00:07:11,870: INFO: common: created directory at: artifacts]
[2025-01-10 16:04:39,388: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:04:39,409: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:04:39,424: INFO: common: created directory at: artifacts]
[2025-01-10 16:04:39,434: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:04:39,440: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:09:17,689: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:09:17,744: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:09:17,761: INFO: common: created directory at: artifacts]
[2025-01-10 16:09:17,821: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:09:17,832: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:15:07,435: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:15:07,472: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:15:07,491: INFO: common: created directory at: artifacts]
[2025-01-10 16:15:07,502: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:15:07,508: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:15:14,669: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:15:14,774: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:15:14,783: INFO: common: created directory at: artifacts]
[2025-01-10 16:15:14,787: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:15:14,800: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:17:10,267: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:17:10,308: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:17:10,323: INFO: common: created directory at: artifacts]
[2025-01-10 16:17:10,328: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:17:10,334: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:17:16,829: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:17:16,846: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:17:16,850: INFO: common: created directory at: artifacts]
[2025-01-10 16:17:16,853: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:17:16,858: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:18:24,834: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:18:24,849: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:18:24,857: INFO: common: created directory at: artifacts]
[2025-01-10 16:18:24,863: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:18:24,876: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:20:08,887: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:20:08,934: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:20:08,945: INFO: common: created directory at: artifacts]
[2025-01-10 16:20:08,956: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:20:08,963: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:22:12,108: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:22:12,123: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:22:12,135: INFO: common: created directory at: artifacts]
[2025-01-10 16:22:12,141: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 16:22:12,144: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 16:37:29,664: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 16:37:29,699: INFO: helpers: Accessing as Calivala-86]
[2025-01-10 16:37:30,067: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-10 16:37:30,351: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 16:37:30,356: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-10 16:37:30,359: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-10 16:37:30,378: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 16:37:30,416: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:37:30,450: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:37:30,463: INFO: common: created directory at: artifacts]
[2025-01-10 16:37:30,464: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 16:37:30,465: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 16:37:40,169: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 16:37:41,737: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 16:37:41,741: INFO: main: *******************]
[2025-01-10 16:37:41,741: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 16:37:41,761: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:37:41,777: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:37:41,781: INFO: common: created directory at: artifacts]
[2025-01-10 16:37:41,782: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 16:37:46,487: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 16:37:48,171: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 16:37:48,172: INFO: main: *******************]
[2025-01-10 16:37:48,172: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 16:37:48,184: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:37:48,196: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:37:48,200: INFO: common: created directory at: artifacts]
[2025-01-10 16:37:48,201: INFO: common: created directory at: artifacts/training]
[2025-01-10 16:45:48,114: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 16:45:48,125: INFO: main: *******************]
[2025-01-10 16:45:48,125: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-10 16:45:48,144: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:45:48,153: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:45:48,159: INFO: common: created directory at: artifacts]
[2025-01-10 16:46:49,057: INFO: common: json file saved at scores.json]
[2025-01-10 16:46:49,060: INFO: common: json file saved at scores.json]
[2025-01-10 16:46:54,742: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-10 16:46:56,010: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpsewfqyau/model/data/model/assets]
[2025-01-10 16:47:53,225: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-01-10 16:47:53,227: INFO: main: *******************]
[2025-01-10 16:47:53,227: INFO: main: >>>>>>> stage CallBacks stage started <<<<<<<]
[2025-01-10 16:47:53,227: ERROR: main: name 'ConfigurationManager' is not defined]
Traceback (most recent call last):
  File "main.py", line 66, in <module>
    callbacks_action.main()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/stage_05_callbacks.py", line 14, in main
    config = ConfigurationManager()
NameError: name 'ConfigurationManager' is not defined
[2025-01-10 16:55:09,241: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 16:55:09,257: INFO: helpers: Accessing as Calivala-86]
[2025-01-10 16:55:09,591: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-10 16:55:10,029: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 16:55:10,039: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-10 16:55:10,040: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-10 16:55:10,049: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 16:55:10,052: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:55:10,063: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:55:10,067: INFO: common: created directory at: artifacts]
[2025-01-10 16:55:10,067: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 16:55:10,068: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 16:55:19,124: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 16:55:20,809: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 16:55:20,809: INFO: main: *******************]
[2025-01-10 16:55:20,809: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 16:55:20,818: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:55:20,834: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:55:20,839: INFO: common: created directory at: artifacts]
[2025-01-10 16:55:20,839: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 16:55:22,754: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 16:55:23,868: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 16:55:23,868: INFO: main: *******************]
[2025-01-10 16:55:23,868: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 16:55:23,878: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:55:23,883: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:55:23,884: INFO: common: created directory at: artifacts]
[2025-01-10 16:55:23,885: INFO: common: created directory at: artifacts/training]
[2025-01-10 16:58:59,562: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 16:58:59,578: INFO: main: *******************]
[2025-01-10 16:58:59,579: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-10 16:58:59,603: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 16:58:59,652: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 16:58:59,661: INFO: common: created directory at: artifacts]
[2025-01-10 16:59:57,074: INFO: common: json file saved at scores.json]
[2025-01-10 16:59:57,086: INFO: common: json file saved at scores.json]
[2025-01-10 17:00:04,188: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-10 17:00:05,403: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpc957nk0h/model/data/model/assets]
[2025-01-10 17:00:52,069: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-01-10 17:00:52,070: INFO: main: *******************]
[2025-01-10 17:00:52,070: INFO: main: >>>>>>> stage CallBacks stage started <<<<<<<]
[2025-01-10 17:00:52,074: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 17:00:52,081: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 17:00:52,087: INFO: common: created directory at: artifacts]
[2025-01-10 17:00:52,087: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 17:00:52,087: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 17:00:52,089: INFO: main: >>>>>>> stage CallBacks stage completed <<<<<<

x==========x]
[2025-01-10 21:30:27,207: INFO: stage_05_callbacks: *******************]
[2025-01-10 21:30:27,208: INFO: stage_05_callbacks: >>>>>>> stage CallBacks stage started <<<<<<<]
[2025-01-10 21:30:27,212: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:30:27,224: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:30:27,232: INFO: common: created directory at: artifacts]
[2025-01-10 21:30:27,234: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 21:30:27,235: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 21:30:27,239: INFO: stage_05_callbacks: >>>>>>> stage CallBacks stage completed <<<<<<

x==========x]
[2025-01-10 21:32:26,212: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 21:32:26,218: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:32:26,225: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:32:26,228: INFO: common: created directory at: artifacts]
[2025-01-10 21:32:26,228: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 21:32:26,228: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:32:34,237: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:32:35,273: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 21:34:06,854: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 21:34:06,861: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:34:06,868: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:34:06,870: INFO: common: created directory at: artifacts]
[2025-01-10 21:34:06,871: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 21:34:06,871: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:34:13,052: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:34:14,235: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 21:36:46,612: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 21:36:46,616: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:36:46,624: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:36:46,627: INFO: common: created directory at: artifacts]
[2025-01-10 21:36:46,628: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 21:36:46,628: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:36:53,070: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:36:54,278: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 21:38:38,833: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 21:38:38,838: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:38:38,845: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:38:38,849: INFO: common: created directory at: artifacts]
[2025-01-10 21:38:38,850: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 21:38:38,850: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:38:45,538: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 21:38:46,509: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 21:49:10,536: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 21:49:10,537: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 21:49:10,548: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:49:10,557: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:49:10,562: INFO: common: created directory at: artifacts]
[2025-01-10 21:49:10,563: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 21:49:11,849: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 21:49:13,212: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 21:53:59,049: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 21:53:59,051: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 21:53:59,062: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:53:59,069: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:53:59,072: INFO: common: created directory at: artifacts]
[2025-01-10 21:53:59,073: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 21:54:00,269: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 21:54:01,571: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 21:55:20,313: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 21:55:20,314: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 21:55:20,324: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 21:55:20,333: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 21:55:20,339: INFO: common: created directory at: artifacts]
[2025-01-10 21:55:20,340: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 21:55:21,565: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 21:55:22,758: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 22:13:59,176: INFO: stage_03_model_training: *******************]
[2025-01-10 22:13:59,177: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 22:13:59,186: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 22:13:59,197: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 22:13:59,203: INFO: common: created directory at: artifacts]
[2025-01-10 22:13:59,203: INFO: common: created directory at: artifacts/training]
[2025-01-10 22:20:44,148: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 22:22:35,057: INFO: stage_03_model_training: *******************]
[2025-01-10 22:22:35,059: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 22:22:35,066: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 22:22:35,088: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 22:22:35,092: INFO: common: created directory at: artifacts]
[2025-01-10 22:22:35,092: INFO: common: created directory at: artifacts/training]
[2025-01-10 22:28:00,366: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 22:57:46,302: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 22:57:46,324: INFO: helpers: Accessing as Calivala-86]
[2025-01-10 22:57:46,749: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-10 22:57:47,047: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 22:57:47,053: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-10 22:57:47,054: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-10 22:57:47,083: INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 22:57:47,104: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 22:57:47,113: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 22:57:47,117: INFO: common: created directory at: artifacts]
[2025-01-10 22:57:47,118: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 22:57:47,119: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 22:57:56,054: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 22:57:58,376: INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 22:57:58,377: INFO: main: *******************]
[2025-01-10 22:57:58,377: INFO: main: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 22:57:58,384: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 22:57:58,388: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 22:57:58,388: INFO: common: created directory at: artifacts]
[2025-01-10 22:57:58,389: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 22:58:00,127: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 22:58:02,038: INFO: main: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 22:58:02,039: INFO: main: *******************]
[2025-01-10 22:58:02,040: INFO: main: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 22:58:02,054: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 22:58:02,057: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 22:58:02,058: INFO: common: created directory at: artifacts]
[2025-01-10 22:58:02,059: INFO: common: created directory at: artifacts/training]
[2025-01-10 23:03:31,284: INFO: main: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 23:03:31,293: INFO: main: *******************]
[2025-01-10 23:03:31,294: INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-10 23:03:31,310: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:03:31,325: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:03:31,337: INFO: common: created directory at: artifacts]
[2025-01-10 23:06:54,758: INFO: common: json file saved at scores.json]
[2025-01-10 23:06:54,765: INFO: common: json file saved at scores.json]
[2025-01-10 23:07:28,478: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-10 23:07:30,283: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpr05kx2th/model/data/model/assets]
[2025-01-10 23:08:22,692: INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-01-10 23:08:22,693: INFO: main: *******************]
[2025-01-10 23:08:22,693: INFO: main: >>>>>>> stage CallBacks stage started <<<<<<<]
[2025-01-10 23:08:22,700: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:08:22,714: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:08:22,719: INFO: common: created directory at: artifacts]
[2025-01-10 23:08:22,721: INFO: common: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-01-10 23:08:22,721: INFO: common: created directory at: artifacts/prepare_callbacks/tensorboard_log_dir]
[2025-01-10 23:08:22,745: INFO: main: >>>>>>> stage CallBacks stage completed <<<<<<

x==========x]
[2025-01-10 23:23:18,972: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 23:23:18,977: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:23:18,987: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:23:19,028: INFO: common: created directory at: artifacts]
[2025-01-10 23:23:19,029: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 23:23:19,029: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 23:23:25,704: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 23:23:27,076: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 23:23:49,543: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 23:23:49,544: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 23:23:49,549: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:23:49,562: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:23:49,567: INFO: common: created directory at: artifacts]
[2025-01-10 23:23:49,569: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 23:23:50,798: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 23:23:52,101: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 23:24:57,208: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 23:24:57,209: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 23:24:57,215: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:24:57,218: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:24:57,219: INFO: common: created directory at: artifacts]
[2025-01-10 23:24:57,220: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 23:24:58,273: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 23:24:59,615: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 23:25:19,140: INFO: stage_03_model_training: *******************]
[2025-01-10 23:25:19,141: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 23:25:19,147: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:25:19,149: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:25:19,150: INFO: common: created directory at: artifacts]
[2025-01-10 23:25:19,150: INFO: common: created directory at: artifacts/training]
[2025-01-10 23:28:40,607: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 23:30:05,584: INFO: stage_03_model_training: *******************]
[2025-01-10 23:30:05,585: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 23:30:05,606: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:30:05,617: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:30:05,621: INFO: common: created directory at: artifacts]
[2025-01-10 23:30:05,622: INFO: common: created directory at: artifacts/training]
[2025-01-10 23:33:01,702: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 23:34:21,640: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 23:34:21,646: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:34:21,652: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:34:21,655: INFO: common: created directory at: artifacts]
[2025-01-10 23:34:21,656: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 23:34:21,656: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 23:34:28,269: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 23:34:29,532: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 23:34:50,566: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 23:34:50,567: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 23:34:50,572: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:34:50,574: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:34:50,575: INFO: common: created directory at: artifacts]
[2025-01-10 23:34:50,575: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 23:34:52,067: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 23:34:53,443: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 23:35:18,562: INFO: stage_03_model_training: *******************]
[2025-01-10 23:35:18,563: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 23:35:18,573: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:35:18,586: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:35:18,591: INFO: common: created directory at: artifacts]
[2025-01-10 23:35:18,594: INFO: common: created directory at: artifacts/training]
[2025-01-10 23:41:52,690: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 23:42:18,055: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-10 23:42:18,061: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:42:18,071: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:42:18,075: INFO: common: created directory at: artifacts]
[2025-01-10 23:42:18,075: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-10 23:42:18,076: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 23:42:24,438: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-10 23:42:25,564: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-10 23:42:49,585: INFO: stage_02_prepare_base_model: *******************]
[2025-01-10 23:42:49,586: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-10 23:42:49,594: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:42:49,603: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:42:49,608: INFO: common: created directory at: artifacts]
[2025-01-10 23:42:49,611: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-10 23:42:52,304: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-10 23:42:53,696: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-10 23:43:17,762: INFO: stage_03_model_training: *******************]
[2025-01-10 23:43:17,763: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-10 23:43:17,769: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:43:17,789: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:43:17,792: INFO: common: created directory at: artifacts]
[2025-01-10 23:43:17,796: INFO: common: created directory at: artifacts/training]
[2025-01-10 23:47:42,487: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-10 23:48:13,056: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 23:48:13,075: INFO: helpers: Accessing as Calivala-86]
[2025-01-10 23:48:13,367: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-10 23:48:13,639: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-10 23:48:13,643: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-10 23:48:13,644: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-10 23:48:13,645: INFO: stage_04_evaluation: *******************]
[2025-01-10 23:48:13,645: INFO: stage_04_evaluation: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-10 23:48:13,671: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-10 23:48:13,682: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-10 23:48:13,687: INFO: common: created directory at: artifacts]
[2025-01-10 23:49:29,832: INFO: common: json file saved at scores.json]
[2025-01-10 23:49:29,858: INFO: common: json file saved at scores.json]
[2025-01-10 23:49:41,282: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-10 23:49:44,273: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpt__re8cg/model/data/model/assets]
[2025-01-10 23:50:50,461: INFO: stage_04_evaluation: >>>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-01-11 00:09:27,072: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2025-01-11 00:09:27,077: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-11 00:09:27,082: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-11 00:09:27,090: INFO: common: created directory at: artifacts]
[2025-01-11 00:09:27,091: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-11 00:09:27,091: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-11 00:09:34,180: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-01-11 00:09:35,414: INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-01-11 00:09:56,813: INFO: stage_02_prepare_base_model: *******************]
[2025-01-11 00:09:56,814: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model started <<<<<<<]
[2025-01-11 00:09:56,823: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-11 00:09:56,825: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-11 00:09:56,826: INFO: common: created directory at: artifacts]
[2025-01-11 00:09:56,827: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-01-11 00:09:58,011: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-01-11 00:09:59,223: INFO: stage_02_prepare_base_model: >>>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-01-11 00:10:19,571: INFO: stage_03_model_training: *******************]
[2025-01-11 00:10:19,572: INFO: stage_03_model_training: >>>>>>> stage Training started <<<<<<<]
[2025-01-11 00:10:19,577: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-11 00:10:19,580: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-11 00:10:19,580: INFO: common: created directory at: artifacts]
[2025-01-11 00:10:19,581: INFO: common: created directory at: artifacts/training]
[2025-01-11 00:14:26,819: INFO: stage_03_model_training: >>>>>>> stage Training completed <<<<<<

x==========x]
[2025-01-11 00:14:52,270: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-11 00:14:52,290: INFO: helpers: Accessing as Calivala-86]
[2025-01-11 00:14:52,626: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC "HTTP/1.1 200 OK"]
[2025-01-11 00:14:52,889: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2025-01-11 00:14:52,895: INFO: helpers: Initialized MLflow to track repo "Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC"]
[2025-01-11 00:14:52,897: INFO: helpers: Repository Calivala-86/Kidney-Disease-Classifciation-Deep-Learning-Project-Mlflow-DVC initialized!]
[2025-01-11 00:14:52,897: INFO: stage_04_evaluation: *******************]
[2025-01-11 00:14:52,898: INFO: stage_04_evaluation: >>>>>>> stage Evaluation stage started <<<<<<<]
[2025-01-11 00:14:52,942: INFO: common: yaml file: config/config.yaml loaded successfully]
[2025-01-11 00:14:52,950: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-11 00:14:52,953: INFO: common: created directory at: artifacts]
[2025-01-11 00:15:47,417: INFO: common: json file saved at scores.json]
[2025-01-11 00:15:47,428: INFO: common: json file saved at scores.json]
[2025-01-11 00:15:54,713: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-01-11 00:15:56,330: INFO: builder_impl: Assets written to: /var/folders/ks/5nym8j9d5_l_9fp2yb3p9q3c0000gn/T/tmpxc_dilsr/model/data/model/assets]
[2025-01-11 00:16:42,899: INFO: stage_04_evaluation: >>>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2025-01-11 20:37:07,000: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.14:8080]
[2025-01-11 20:37:07,001: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-01-11 20:40:07,970: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:40:07] "[33mGET /eventos HTTP/1.1[0m" 404 -]
[2025-01-11 20:40:08,400: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:40:08] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-01-11 20:42:36,787: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:42:36] "GET / HTTP/1.1" 200 -]
[2025-01-11 20:42:48,047: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 40, in predictRoute
    result = clApp.classifier.predict()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/prediction.py", line 13, in predict
    model = load_model(os.path, join("artitacts", "training", "model.h5"))
NameError: name 'join' is not defined
[2025-01-11 20:42:48,072: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:42:48] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-01-11 20:43:17,949: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:43:17] "GET / HTTP/1.1" 200 -]
[2025-01-11 20:44:02,151: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 40, in predictRoute
    result = clApp.classifier.predict()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/prediction.py", line 13, in predict
    model = load_model(os.path, join("artitacts", "training", "model.h5"))
NameError: name 'join' is not defined
[2025-01-11 20:44:02,218: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:44:02] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-01-11 20:47:30,134: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:47:30] "GET / HTTP/1.1" 200 -]
[2025-01-11 20:48:01,030: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 40, in predictRoute
    result = clApp.classifier.predict()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/prediction.py", line 13, in predict
    model = load_model(os.path.join("artitacts", "training", "model.h5"))
NameError: name 'join' is not defined
[2025-01-11 20:48:01,043: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:48:01] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-01-11 20:48:20,732: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:48:20] "GET / HTTP/1.1" 200 -]
[2025-01-11 20:49:40,552: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.14:8080]
[2025-01-11 20:49:40,553: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-01-11 20:49:53,244: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:49:53] "GET / HTTP/1.1" 200 -]
[2025-01-11 20:50:04,938: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 40, in predictRoute
    result = clApp.classifier.predict()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/prediction.py", line 13, in predict
    model = load_model(os.path.join("artitacts", "training", "model.h5"))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/saving/legacy/save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at artitacts/training/model.h5
[2025-01-11 20:50:04,952: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:50:04] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-01-11 20:58:36,175: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 20:58:36] "GET / HTTP/1.1" 200 -]
[2025-01-11 21:27:03,988: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.14:8080]
[2025-01-11 21:27:03,990: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-01-11 21:27:17,650: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 21:27:17] "GET / HTTP/1.1" 200 -]
[2025-01-11 21:27:31,348: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 40, in predictRoute
    result = clApp.classifier.predict()
  File "/Users/fernandoecumbi/Desktop/IT course/Kidney-Disease-Classifciation-Deep-Learning-Project/src/cnnClassifier/pipeline/prediction.py", line 14, in predict
    model = load_model(os.path.join("artitacts", "training", "model.h5"))
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/tensorflow/python/keras/saving/save.py", line 205, in load_model
    return saved_model_load.load(filepath, compile, options)
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/tensorflow/python/keras/saving/saved_model/load.py", line 122, in load
    meta_graph_def = loader_impl.parse_saved_model(path).meta_graphs[0]
  File "/Applications/anaconda3/envs/kidney/lib/python3.8/site-packages/tensorflow/python/saved_model/loader_impl.py", line 116, in parse_saved_model
    raise IOError(
OSError: SavedModel file does not exist at: artitacts/training/model.h5/{saved_model.pbtxt|saved_model.pb}
[2025-01-11 21:27:31,372: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 21:27:31] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2025-01-11 21:28:26,737: INFO: _internal: 127.0.0.1 - - [11/Jan/2025 21:28:26] "GET / HTTP/1.1" 200 -]
